tests/languages/java-file/Makefile

```diff
@@ -0,0 +1,36 @@
+include ../../common.mk
+
+APP_ROOT=app
+PROG=MainApp
+
+DISK_IMAGE=sgxlkl-java-fs.img
+IMAGE_SIZE=400M
+
+ifeq ($(SGXLKL_VERBOSE),)
+SGXLKL_ENV=\
+   SGXLKL_VERBOSE=1 SGXLKL_KERNEL_VERBOSE=1 SGXLKL_TRACE_LKL_SYSCALL=0 SGXLKL_TRACE_SIGNAL=0
+else
+SGXLKL_ENV=
+endif
+
+.DELETE_ON_ERROR:
+.PHONY: run run-hw run-sw clean
+
+$(APP_ROOT)/%.class: $(APP_ROOT)/%.java
+	javac -source 1.8 -target 1.8 $(APP_ROOT)/*.java
+
+$(DISK_IMAGE): $(APP_ROOT)/$(PROG).class
+	${SGXLKL_DISK_TOOL} create --size=${IMAGE_SIZE} --alpine="iputils iproute2 unzip libstdc++ openjdk8-jre nss" --copy=./${APP_ROOT}/ ${DISK_IMAGE}
```

Comment:
Could you use a Dockerfile instead of `--alpine`? With a Dockerfile you can then compile inside Docker. I'd like to move over to Docker images for all new tests and gradually for existing ones as well.

Reply:
Implemented using Dockerfile and verified it is working properly.